src/workerd/api/crypto-impl-asymmetric.c++

```diff
@@ -702,6 +730,12 @@ kj::OneOf<jsg::Ref<CryptoKey>, CryptoKeyPair> CryptoKey::Impl::generateRsa(
       publicExponent.size(), nullptr), InternalDOMOperationError, "Error setting up RSA keygen.");
 
   auto rsaPrivateKey = OSSL_NEW(RSA);
+  // TODO: boringssl silently uses (modulusLength & ~127) for the key size, i.e. it rounds down to
```

Comment:
Nit: We'll typically use a style like `TODO(soon)` or `TODO(conform)` etc. Non-blocking tho